test/Microsoft.ML.Tests/Transformers/ConvertTests.cs

```diff
@@ -98,7 +98,7 @@ public void TestConvertWorkout()
                   new TestPrimitiveClass()
                 {
                     AA = new []{"0", "1"},
-                    AB = new []{false, true},
+                    AB = new []{true, false},
```

Comment:
>true, false} [](start = 32, length = 12)

p.s. I modified the order just to ensure that the 2nd row is different from the 1st row above.